xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -1188,11 +1194,14 @@ public void cdsResponseWithMatchingResource() {
     assertThat(clusterUpdate.isEnableLrs()).isEqualTo(false);
 
     // Management server sends back another CDS response updating the requested Cluster.
+    UpstreamTlsContext testUpstreamTlsContext =
```

Comment:
Ideally, I would recommend writing a separate test method covering the usage of this field, instead of incorporating into an existing one. It would be hard to see if we did so or not in the future.

But, I would also accept this, as `upstreamTlsContext` is optional.

Reply:
I will make a separate test - I did think about it.